packages/graphql-extensions/src/index.ts

```diff
@@ -31,27 +31,46 @@ export interface GraphQLResponse {
   extensions?: object;
 }
 
+export interface RequestDidStartOpts<TContext> {
```

Comment:
Thanks for breaking these interfaces out!

Can we be more explicit than `*Opts` and just call these `*Options`?  Unless there's a pattern here that's being followed that I'm missing?

Reply:
No reason. Updated